src/window.rs

```diff
@@ -624,6 +624,10 @@ impl Window {
 
     /// Gets the window's current fullscreen state.
     ///
+    /// If `None` in `Fullscreen::Borderless` was returned it means
+    /// that the window is in a `Fullscreen` state, however there's
+    /// no monitor information available.
```

Comment:
This seems a bit problematic, because we seem to conflate the concepts of "no monitor information available" and "current / primary monitor". Like a user doesn't request fullscreen on no monitor, that doesn't make sense.

Reply:
I mean what should I describe here then? It can clearly be returned with `None` on GNOME(Wayland) :P